src/utils/test-head.js

```diff
@@ -50,7 +50,7 @@ function commonLog(msg: string, data: any = {}) {
 function makeSource(name: string, props: any = {}) {
   return {
     id: name,
-    loadedState: "loaded",
+    loadedState: "unloaded",
```

Comment:
changed this, because we more often need the unloaded state than the loaded state

Reply:
makes sense

Reply:
nice

Reply:
nice